daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -721,6 +783,7 @@ func (devices *DeviceSet) setupBaseImage() error {
 		return err
 	}
 
+	devices.saveBaseDeviceUUID(info)
```

Comment:
Not critical, but wouldn't hurt
On Jun 16, 2015 17:40, "David Calavera" <notifications@github.com> wrote:

> In daemon/graphdriver/devmapper/deviceset.go
> <https://github.com/docker/docker/pull/13896#discussion_r32574282>:
>
> > @@ -721,6 +783,7 @@ func (devices *DeviceSet) setupBaseImage() error {
> >  		return err
> >  	}
> >
> > +	devices.saveBaseDeviceUUID(info)
>
> is the error returned by saveBaseDeviceUUID ignored here on purpose?
> should we return it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/13896/files#r32574282>.
>